docs/index.asciidoc

```diff
@@ -346,7 +346,7 @@ If "read" is specified then the following other settings are ignored:
 . `start_position` (files are always read from the beginning)
 . `close_older` (files are automatically 'closed' when EOF is reached)
 
-If "read" is specified then the following settings are heeded:
+If "read" is specified then the following settings are needed:
 
```

Comment:
Nice find, @kristoffeys. After researching further, I think the original author intended to use "heeded" to indicate that those settings would be _noticed_ when the "read" option is used. "Heeded" isn't a word that people frequently use in conversation, and I think we can make it better. 

I'm not sure that "needed" is the right word, because to me that implies that the following settings are  'required.'  Only one of those options can be used. 

What do you think about using "considered?"  As in "If 'read' is specified, then. the following settings are considered?" Please let me know what you think.

Reply:
I'm not a native English speaker & didn't even know heeded was a word. Are all of those options optional? 
If so "considered" is indeed an option. Another option might be: `If "read" is specified then the following settings can be used`

Seems better than needed as that indeed implies that they are mandatory

Reply:
If the user specifies `read`, then the user can use those settings to further control the behavior. 

"Considered" and "can be used" are both good options.  You are the author of this PR, so you get to choose which you prefer.  :-)

Reply:
I've updated the term, thanks! :)